README.md

```diff
@@ -47,34 +81,59 @@ These sections describe requirements for using this module.
 
 The following dependencies must be available:
 
-- [Terraform][terraform] v0.12
-- [Terraform Provider for GCP][terraform-provider-gcp] plugin v2.0
+* [Terraform][terraform] v0.12
```

Comment:
We should change the cookiecutter [versions.tf](https://github.com/terraform-google-modules/terraform-google-group/blob/xingao/versions.tf) to allow 0.13 and require [TPGB](https://github.com/hashicorp/terraform-provider-google-beta/blob/master/CHANGELOG.md#3290-july-06-2020) `>= 3.29.0, <4.0.0`

Reply:
Done